src/toil/wdl/wdl_functions.py

```diff
@@ -287,7 +287,7 @@ def process_single_outfile(wdl_file: WDLFile, fileStore, workDir, outDir) -> WDL
     return WDLFile(file_path=output_file, file_name=preserveThisFilename, imported=True)
 
 
-def process_outfile(f, fileStore, workDir, outDir):
+def process_outfile(f: Any, fileStore: AbstractFileStore, workDir: str, outDir: str) -> Any:
```

Comment:
```suggestion
def process_outfile(f: Union [WDLFile, List[Any], WDLPair, Dict[Any, Any], int, str, bool, float], fileStore: AbstractFileStore, workDir: str, outDir: str) -> Union[WDLFile, List[Any], WDLPair, Dict[Any, Any], int, str, bool, float]:
```

Reply:
Maybe that giant union should be a named type variable? Why are all those types and those types specifically allowed here?